components/common/infinite_scroll.test.tsx

```diff
@@ -17,10 +18,10 @@ describe('/components/common/InfiniteScroll', () => {
         pageNumber: 1,
     };
 
-    let wrapper;
+    let wrapper: ReactWrapper<any, any, InfiniteScroll>;
 
     beforeEach(() => {
-        wrapper = mountWithIntl(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>);
+        wrapper = mountWithIntl(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>) as unknown as ReactWrapper<any, any, InfiniteScroll>;
```

Comment:
I think mountWithIntl should also support that type parameter as well. I'm not 100% sure though
```suggestion
        wrapper = mountWithIntl<InfiniteScroll>(<InfiniteScroll {...baseProps}><div/></InfiniteScroll>);
```

Reply:
Nope